Resources/Server Info/RP_Rules.txt

```diff
@@ -0,0 +1,103 @@
+﻿
+[color=#ff0000]You must be 16 or older to play. Users under 16 will be banned immediately and be unbanned when they are old enough.[/color]
+[color=#a4885c]The[/color] [color=#ffd700]Golden[/color] [color=#a4885c]Rule.[/color] Admins can disregard any and all of these rules if they deem it in the best interest of the current round, server, and/or community at large. They will of course be held fully accountable for their actions if they exercise this privilege.
```

Comment:
```suggestion
[color=#a4885c]The[/color] [color=#ffd700]Golden[/color] [color=#a4885c]Rule.[/color] Admins can disregard any and all of these rules if they deem it in the best interest of the current round, server, and/or community at large. They will of course be held fully accountable for their actions if they exercise this privilege.
    - An admin's word is law—follow it, don't argue it. If you think an admin is acting unfairly or otherwise abusing their position, you can file a complain on the forums later.
```